check-log/lib/check-log.go

```diff
@@ -247,11 +247,14 @@ func (opts *logOpts) searchLog(ctx context.Context, logFile string) (int64, int6
 		return 0, 0, "", nil
 	}
 	stateFile := getStateFile(opts.StateDir, logFile, opts.origArgs)
-	skipBytes, inode := int64(0), uint(0)
+	skipBytes, inode, isFirstCheck := int64(0), uint(0), false
 	if !opts.NoState {
 		s, err := getBytesToSkip(stateFile)
 		if err != nil {
-			return 0, 0, "", err
+			if err != errValidStateFileNotFound {
```

Comment:
`getBytesToSkip` can return `errValidStateFileNotFound` only if `getBytesToSkipOld` is called.